example/tests/integration/test_polymorphism.py

```diff
@@ -168,27 +168,43 @@ def test_polymorphism_relations_update(single_company, research_project_factory,
         "type": "researchProjects",
         "id": research_project.pk
     }
-    response = client.put(reverse("company-detail", kwargs={'pk': single_company.pk}),
-                          data=content)
+    response = client.patch(reverse("company-detail", kwargs={'pk': single_company.pk}),
+                            data=content)
     assert response.status_code == 200
     content = response.json()
     assert content["data"]["relationships"]["currentProject"]["data"]["type"] == "researchProjects"
     assert int(content["data"]["relationships"]["currentProject"]["data"]["id"]) == \
         research_project.pk
 
 
-def test_invalid_type_on_polymorphic_relation(single_company, research_project_factory, client):
+def test_polymorphism_relations_put_405(single_company, research_project_factory, client):
+    # the preceding had been incorrectly using PUT instead of PATCH. This confirms PUT now fails.
```

Comment:
this comment is not necessary as the test method already explains what it does. Better to be removed.